zone/corpse.cpp

```diff
@@ -622,7 +622,7 @@ void Corpse::CalcCorpseName()
 	EntityList::RemoveNumbers(name);
 	char tmp[64];
 	if (m_is_player_corpse) {
-		snprintf(tmp, sizeof(tmp), "'s corpse%d", GetID());
+		snprintf(tmp, sizeof(tmp), "`s corpse%d", GetID());
```

Comment:
Do we even need the if (m_is_player_corpse) condition if we make them both backtics?

Reply:
For sure.  Holding off until we can see if we need this backtick hack at all, but yes.

Reply:
Removed this aspect (PC corpses) from the fix entirely as it created too many side effects.